SQL/New_patches/2019-03-20_add_instrument_hash.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE `instrument_schema` (
```

Comment:
I think we should add the actual schema as well... the case being that we cannot stop all data-entry in the event that the server is offline, unreachable or corrupted. plus it would avoid an external call for each instrument load/save/build...

Reply:
sure, that's shouldn't be a problem as long as we store only the final, complete, and validated schema